core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -306,6 +306,7 @@ public String toString() {
           .setDefaultValue(String.format("${%s}/conf", Name.HOME))
           .setDescription("The directory containing files used to configure Alluxio.")
           .setIgnoredSiteProperty(true)
+          .setIsHidden(true)
```

Comment:
for these 3 unique properties that should be set through env, can the env var be mentioned somewhere in the property definition, ex. in its description?
```
.setDescription("The directory containing files used to configure Alluxio. This should only be set via the environment variable ALLUXIO_CONF_DIR when invoking startup scripts and never be set as a java opt or in alluxio-site.properties.")
```

Reply:
done